com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,3 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.chartboost.mediationsdk.unity">
+    package="com.chartboost.mediation.unity">
```

Comment:
`nit:` We may be lucky and not run into a name conflict. But, if we do:

Thoughts on having the package name as:
```suggestion
    package="com.chartboost.mediation.unitybridge">
```